examples/analog_hamiltonian_simulation/09_Noisy_quantum_dynamics_for_Rydberg_atom_arrays.ipynb

```diff
@@ -0,0 +1,1691 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.                        mydict[key] = [RabiCorrection(rampTime=Decimal(str(t)), rabiCorrection=Decimal(str(v))) for (t, v) in value]
</pre><p>This seems like it assumes that every list in the performance model is of type <code>RabiCorrection</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Indeed, I have assumed that, and the assumption holds for the current version of the schema.